bitcoind-rpc/src/main/scala/org/bitcoins/rpc/client/common/BitcoindRpcClient.scala

```diff
@@ -350,10 +352,10 @@ sealed trait BitcoindVersion
 object BitcoindVersion extends StringFactory[BitcoindVersion] {
 
   /** The newest version of `bitcoind` we support */
-  val newest: BitcoindVersion = V22
+  val newest: BitcoindVersion = V23
```

Comment:
Can you try reverting this and see if the `lndTest` issues go away? It is probably worth looking at the lnd release notes to see if they fully support `v23` of bitcoind

Reply:
Hmm @benthecarman any educated guesses from you? We haven't been merging a ton of PRs lately, but this does seem like more failures than normal. Ben recently merged some work to fix one of the consistently failing test cases in #4361

Reply:
I have been running lnd v0.14.3 with bitcoind v23 with no issues. Looks like CI is passing now